snapcraft.yaml

```diff
@@ -304,30 +290,37 @@ parts:
     prime:
       - -etc/fonts
       - -var
-      - -include
-      - -share/man
-      - -share/doc
+      - -usr/include
+      - -usr/lib/*/pkgconfig
+      - -usr/lib/*/libqpdf.a
+      - -usr/lib/cmake
       - -share/lintian
       - -usr/share/fonts
       - -usr/share/man
       - -usr/share/doc
       - -usr/share/doc-base
       - -usr/share/lintian
-      - -usr/lib/libqpdf.a
 
   ghostscript:
     # use GitHub mirror of GhostPDL here, as original GIT, self-hosted by
     # Artifex is not supported by ubuntu/desktop-snaps
     #source: https://git.ghostscript.com/ghostpdl.git
     source: https://github.com/ArtifexSoftware/ghostpdl.git
     source-type: git
-    source-tag: 'ghostpdl-10.05.1rc1_test002'
+    source-tag: 'ghostpdl-10.05.1'
     source-depth: 1
 # ext:updatesnap
 #   version-format:
 #     format: "ghostpdl-%M.%m.%R"
 #     lower-than: '11'
 #     no-9x-revisions: true
+# The repo is actually ghostpdl and not only Ghostscript, so a wider
+# variety of print data renderers/interpreters/converters. One of
+# them, gpdl, does not build with our ./configure settings but we
+# actually do not need it. So delete its source directory to skip
+# it.
+# We also delete the source directories pcl and xps to not build
+# further storage-space-consuming tools.
     plugin: autotools
```

Comment:
The comment needs to get adapted to the different method you are using. I had deleted source dirs, you have added command line options to `./configure`.